sources/packages/api/src/route-controllers/application/application.controller.ts

```diff
@@ -376,13 +376,16 @@ export class ApplicationController extends BaseController {
   /**
    * Get program year details for the application.
    * @param applicationId application id to be updated.
+   * @query includeInActivePY, if includeInActivePY is true,
+   * then consider both active and inactive program year.
    * @returns program year details of the application
    */
   @AllowAuthorizedParty(AuthorizedParties.student)
   @Get(":applicationId/program-year")
   async programYearOfApplication(
     @UserToken() userToken: IUserToken,
     @Param("applicationId") applicationId: number,
+    @Query("includeInActivePY") includeInActivePY?: boolean,
```

Comment:
This Query argument is repeated many times. Can you declare a constant and use?

Reply:
mm, I seriously don't think, there is a use of declaring const for params and query variable, do you have any strong reason @dheepak-aot, pls let me know, if I am missing  anything

Reply:
The reason I mentioned here is reason of a constant, because the same text is repeated several times and declaring it once and using constant will minimize the probability of typo and if at all it needs to be changed, just modifying at one place. 
I couldn't imagine any downside of doing it.
Let me know if I am missing anything.